client/web/compose/src/components/ModuleFields/Editor/Record.vue

```diff
@@ -109,12 +107,10 @@
           option-value="recordID"
```

Comment:
Vue-select does not have this by default, and if I'd implement it, it would be more complicated than actually just putting a reduce because I'd have to go through all of the options and remap them. Or is this what you are suggesting?